resources/config_configuration_recorders.go

```diff
@@ -83,16 +120,57 @@ func fetchConfigConfigurationRecorders(ctx context.Context, meta schema.ClientMe
 	if err != nil {
 		return err
 	}
-	res <- resp.ConfigurationRecorders
+	if len(resp.ConfigurationRecorders) == 0 {
+		return nil
+	}
+	names := make([]string, len(resp.ConfigurationRecorders))
+	for i, configurationRecorder := range resp.ConfigurationRecorders {
+		names[i] = *configurationRecorder.Name
+	}
+	status, err := c.Services().ConfigService.DescribeConfigurationRecorderStatus(ctx, &configservice.DescribeConfigurationRecorderStatusInput{
+		ConfigurationRecorderNames: names,
+	})
+	if err != nil {
+		return err
+	}
+	for _, configurationRecorder := range resp.ConfigurationRecorders {
+		var configurationRecorderStatus types.ConfigurationRecorderStatus
+		for _, s := range status.ConfigurationRecordersStatus {
+			if s.Name == configurationRecorder.Name {
+				configurationRecorderStatus = s
+				break
+			}
+		}
+		res <- configurationRecorderWrapper{
```

Comment:
Can we add an `exists` type field? That way we can have positive confirmation of the record in the database compared to having to handle if the record is missing for whatever reason (not in config, permissions issue, etc)